src/filters/filters.jl

```diff
@@ -429,6 +429,54 @@ function Base.convert(::Type{I}, ::Type{F}) where {I<:Integer,F<:Filter}
     Base.convert(I, filterid(F))
 end
 
+"""
+    EXTERNAL_FILTER_JULIA_PACKAGES
+
+Maps filter id to the Julia package name that contains the filter.
+"""
+const EXTERNAL_FILTER_JULIA_PACKAGES = Dict{API.H5Z_filter_t,String}([
+    32008 => "H5Zbitshuffle",
+    32001 => "H5Zblosc",
+    307   => "H5Zbzip2",
+    32004 => "H5Zlz4",
+    32015 => "H5Zzstd",
+])
+
+"""
+Error if all filters in a filter pipeline are not available.
+"""
+function ensure_filters_available(f::FilterPipeline)
+    if !API.h5p_all_filters_avail(f.plist)
+        nfilters = length(f)
+        for i in 1:nfilters
+            filter::UnknownFilter = getindex(f, UnknownFilter, i)
+            filter_id = filterid(filter)
+            filter_name = filtername(filter)
+            if (!API.h5z_filter_avail(filter_id))
```

Comment:
@mkitti shouldn't the formatter have caught the superfluous parens in the `if` statement

Reply:
Might have to see if that is configurable or not:
https://domluna.github.io/JuliaFormatter.jl/stable/